src/Particles/babylon.particleSystem.ts

```diff
@@ -184,6 +222,10 @@
 
                         this.gravity.scaleToRef(this._scaledUpdateSpeed, this._scaledGravity);
                         particle.direction.addInPlace(this._scaledGravity);
+
+                        if (this.cellSize && this._animationStarted) {
+                            particle.updateCellIndex(deltaTime);

```

Comment:
This is where I was unclear. Really sorry about that (Remember the good old times when we were discussing about PR on the UWP toolkit :D)
There should be no deltatime. the cellIndex should be like the color or the direction. It should have a step per frame (or update). It should be float to accumulate value and obviously only the integer part has to be sent to the vertex buffer.